javascript/welcomeFrm.js

```diff
@@ -4,10 +4,12 @@ const EMAIL_REGEX = /[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-
 // Regex found at https://stackoverflow.com/questions/4338267/validate-phone-number-with-javascript
 const PHONE_REGEX = /^[\+]?[(]?[0-9]{3}[)]?[-\s\.]?[0-9]{3}[-\s\.]?[0-9]{4,6}$/im;
 
-//Regex found at https://stackoverflow.com/questions/13194322/php-regex-to-check-date-is-in-yyyy-mm-dd-format
-const BIRTHDAY_REGEX =/^[0-9]{4}-(0[1-9]|1[0-2])-(0[1-9]|[1-2][0-9]|3[0-1])$/;
+// Regex found at https://stackoverflow.com/questions/13194322/php-regex-to-check-date-is-in-yyyy-mm-dd-format
+// const BIRTHDAY_REGEX =/^[0-9]{4}-(0[1-9]|1[0-2])-(0[1-9]|[1-2][0-9]|3[0-1])$/;
+// Regex found at https://forum.codeigniter.com/thread-68064.html
+const BIRTHDAY_REGEX = /^(\d{2})\/(\d{2})\/(\d{4})$/;
```

Comment:
This is a very simple regex meaning I would be able to type 99/99/9999 and it would pass validation as a correct date